osh/hub/scan/messaging.py

```diff
@@ -26,11 +26,11 @@
 
 
 class UMBSender(proton.handlers.MessagingHandler):
-    def __init__(self, key, msg):
+    def __init__(self, key, msg, topic):
```

Comment:
I think this should be named `topic_prefix` for consistency (and to make it obvious that it will be extended).

Reply:
Just changed it!